pages/manage/productions/index.vue

```diff
@@ -56,7 +56,7 @@
   import DeleteModal from '@/components/productions/index/delete-modal';
   import { parseId as parseEventId } from '@/functions/events';
   import PubWrapper from '@/publiq-ui/pub-wrapper';
-  import pubH1, { Heading1Variant } from '@/publiq-ui/pub-h1';
+  import pubH1 from '@/publiq-ui/pub-h1';
```

Comment:
Nit: `pubH1` should be capitalized.
```suggestion
  import PubH1 from '@/publiq-ui/pub-h1';
```
If you change it, make sure to change the occurence in `components` as well down below.